src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Automation/Peers/AutomationPeer.cs

```diff
@@ -2312,6 +2377,16 @@ private static void Initialize()
             {
                 s_propertyInfo[AutomationElementIdentifiers.PositionInSetProperty.Id] = new GetProperty(GetPositionInSet);
             }
+
+            // HeadingLevel is an RS4+ UIAutomation property.  This will be NULL on other operating system versions.
```

Comment:
Remove this comment (or add a similar comment to all the nearby lines).  It gives the impression that HeadingLevel is different from the other properties, which it isn't.

Reply:
Done